python/ql/lib/change-notes/2022-10-04-api-subscript-nodes.md

```diff
@@ -3,4 +3,4 @@ category: minorAnalysis
 ---
 * Fixed labels in the API graph pertaining to definitions of subscripts. Previously, these were labelled by `getMember` rather than `getASubscript`.
 * Added `API::SubscriptReadNode` and `API::SubscriptWriteNode` with convenience predicates to obtain the index and, in the case of a write, the written value.
-* Added convenience predicates to obtain a subscript at a specific index, when the index happens to be a statically known string.
+* Added convenience predicates `getMember("key")` to obtain a subscript at a specific index, when the index happens to be a statically known string.
```

Comment:
I think it should be this? :confused: 

```suggestion
* Added convenience predicates `getSubscript("key")` to obtain a subscript at a specific index, when the index happens to be a statically known string.
```

Reply:
Wow, yes, now I understand your confusion...maybe lack of refreshes from me drove this concurrency issue..